src/customTypes.ts

```diff
@@ -0,0 +1,96 @@
+import type { Events, Message } from "discord.js";
+
+export type Command = {
+  name: string;
+  command: string | RegExp;
+  description: string;
+  showInHelp: boolean;
+  match: (message: Message) => boolean;
+  execute: (message: Message) => MaybePromiseVoid;
+};
+
+export const CommandGuard = (object: object) =>
+  "match" in object
+  && "execute" in object;
+
+export type BotEvent = {
+  type: Events;
+  once?: boolean;
+  // deno-lint-ignore no-explicit-any
+  execute: (...args: any[]) => void;
+  // These types and parameters differ wildly, I also don't want to use any but I have no choice
+};
+
+export const BotEventGuard = (object: object) =>
+  "type" in object && "execute" in object;
```

Comment:
```suggestion
export const isBotEvent = (object: object) => "type" in object && "execute" in object;
```

Use consistent naming, according to your deno linting rules.